sass_processor/management/commands/compilescss.py

```diff
@@ -49,7 +50,7 @@ def __init__(self):
         self.parser = None
         self.template_exts = getattr(settings, 'SASS_TEMPLATE_EXTS', ['.html'])
         self.sass_output_style = getattr(settings, 'SASS_OUTPUT_STYLE',
-            'nested' if settings.DEBUG else 'compressed')
+                                         'nested' if settings.DEBUG else 'compressed')
```

Comment:
My code formatter did this, I can fix this in a follow-up commit along with the other odd ones to make them look less _odd_. Perhaps to something along these lines:

```python
self.sass_output_style = getattr(
    settings,
    'SASS_OUTPUT_STYLE',
    'nested' if settings.DEBUG else 'compressed'
)
```